src/extensions/nativeExtensions/image/ImageForm.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
```

Comment:
Could you please add indentation here?

![image](https://user-images.githubusercontent.com/13103045/80911374-852c8e80-8d3e-11ea-8137-59144a032b02.png)

Reply:
I don't understand what you are asking. By intention do you mean instructions for the user?

Reply:
I mean `margin`/`padding` for popup's content 😄

Reply:
Yep, done! I misread indentation for intention :man_facepalming: